app-starter/static/app-conf-minimal.json

```diff
@@ -1,6 +1,9 @@
 {
 
-  "baseColor": "green darken-3",
+  "theme": {
```

Comment:
Is it possible to not provide the `theme` property at all?
As fallback a default color scheme could be chosen ...

Reply:
Yes, I was thinking about that too. Wegue will fallback to it's red theme if no theme is provided :+1:

Reply:
In the latest commit, Wegue will fallback to it's default theme configuration.

If there is a `colorTheme` in the config, it will be merged with the default (shallow merge). This means that users will be able to override the property `dark` and `themes`, but not the inner properties of `themes`.